classification_analysis.ipynb

```diff
@@ -1479,7 +1536,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot_f_scores('android')"
+    "plot_f_scores('android', criterion='distance')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb912632",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def export_confusion_matrix(matrix: pd.DataFrame, output_dir: Path):\n",
+    "    pass"
```

Comment:
haven't you implemented this yet?

Reply:
Yes, it is implemented. I will push the finished code.